src/styl/responsive/mobile.styl

```diff
@@ -27,6 +27,24 @@ p
   p
     hyphens: none
 
+#honeypot
```

Comment:
I wonder if the trigger should be a class so they could potentially have the text and effect in multiple locations?